src/dropdown.ts

```diff
@@ -1808,14 +1873,14 @@ export default class Dropdown extends LitElement implements FormControl {
   }
 
   async #onInputInput(event: Event) {
-    // Allowing the event to propagate would break things for consumers, who
-    // expect "input" events only when an option is selected or deselected.
+    // Allowing the event to propagate would break things for consumers or at least
+    // confuse them. They expect "input" events only when an option is selected or
+    // deselected.
     event.stopPropagation();
 
     this.open = true;
-    this.isShowSingleSelectIcon = false;
 
-    if (this.#inputElementRef.value?.value) {
+    if (this.#inputElementRef.value) {
```

Comment:
This change doesn't fix it anything. It just makes it so `this.inputValue` is always in sync with the input field.